src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Methods.jl

```diff
@@ -199,6 +199,67 @@ function components(X::AbsSpec)
   return result
 end
 
+########################################################################
+# Base change and reduction modulo p
+########################################################################
+
+@doc raw"""
+    change_base(phi::Any, X::AbsSpec)
+
+For an affine scheme `X` over a `base_ring` ``𝕜`` and a morphism 
+``φ : 𝕜 → 𝕂`` this computes ``Y = X × Spec(𝕂)`` and returns a pair 
+`(Y, psi)` where `psi` is the canonical map ``Y → X``.
+"""
+function change_base(phi::Any, X::AbsSpec)
```

Comment:
No need to annotate `::any`  .

Reply:
I know. But it makes it clear that the programmer really meant this and was not just too lazy to spell out the annotation and then later forgot about it.

Reply:
Then you could document, that it also takes any kind of function on the coefficients?